Wabbajack/View Models/SlideShow.cs

```diff
@@ -82,23 +82,23 @@ public SlideShow(InstallerVM appState)
                 {
                     if (modList?.SourceModList?.Archives == null)
                     {
-                        return Observable.Empty<NexusDownloader.State>()
-                            .ToObservableChangeSet(x => x.ModID);
+                        return Observable.Empty<AbstractMetaState>()
+                            .ToObservableChangeSet(x => x.Name);
```

Comment:
This will result in any two mods with the same name colliding (and replacing one another, I think?).    Probably not an issue in reality, I imagine, but it's a theoretical issue.   Name is probably a fine enough key for real life.

Two ways to solve the theoretical problem if we want to:
1)  Find a way to uniquely identify all mods (before when it was nexus, the ID was enough.  With cross-site mods, though, no longer)
2)  I think we could actually just remove the `Cache`/`Dictionary`/`Key` concepts and just use list mechanics.  Basically.. we assume the modlist doesn't have duplicate entries.  Swap to `.ToObservableChangeSet()` with no key specified.
Would have to try and test.